unified_planning/test/examples/realistic.py

```diff
@@ -886,4 +886,549 @@ def get_example_problems():
     robot_static_fluents_duration = Example(problem=problem, plan=plan)
     problems["robot_with_static_fluents_duration"] = robot_static_fluents_duration
 
+    ####################################################DEPOT####################################################
```

Comment:
OK, but all these problems are **not** multi-agent...

Reply:
I think an example like this can be moved to the https://github.com/aiplan4eu/planning-test-cases repo.
Here we are more interested in small, unit cases to automatically check the code at each commit.